Mac-App/Mac-App/BackendSubscribers/registerSubscribers.swift

```diff
@@ -11,7 +11,7 @@ import Frontend
 
 func registerSubscribers() {
     BackendAPI.subscribe(listViewTransformer.stateObserver) { $0.select(DependencyTree.Data.init) }
-    BackendAPI.subscribe(projectDetailsTransformer.stateObserver) { $0.select(ProjectDetails.Data.init) }
+    BackendAPI.subscribe(projectDetailsTransformer.stateObserver) { $0.select(ProjectDetails.State.init) }
```

Comment:
This is where I think the `State` naming aligns better with the rest of the system.